packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/utils/forEachSelectedCell.ts

```diff
@@ -10,18 +10,13 @@ import { VTable } from 'roosterjs-editor-dom';
  */
 export function forEachSelectedCell(vTable: VTable, callback: (cell: VCell) => void): number {
     let selectedCells = 0;
+    vTable.selection = normalizeTableSelection(vTable.selection, vTable.table);
```

Comment:
Suggest not change VTable for this function since it looks like just a reader, but not a writer.

You can do either
1. When set selection, already normalize it. or
2. Save the normalize result to a local var